ros/rolling/patch/etc/ros/rosdep/sources.list.d/10-ms-iot.list

```diff
@@ -0,0 +1,2 @@
+# os-specific listings first
+yaml file://localhost/c:\opt\ros\rolling\x64\etc\ros\rosdep\windows.yaml windows
```

Comment:
Is this correct? It looks really odd. Shouldn't it be something like `file:///c:/`

Reply:
Both work in Python. But I changed it to `file:///` which seems to be more commonly seen.